seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -311,28 +312,74 @@ def test_validate_anvil_vcf(self, mock_file_iter, mock_file_exist, mock_utils_lo
+REQUEST_BODY_ADD_DATA['vcfSamples'] = PROJECT1_SAMPLES
+
+REQUEST_BODY_ADD_DATA2 = deepcopy(REQUEST_BODY)
+REQUEST_BODY_ADD_DATA2['vcfSamples'] = PROJECT2_SAMPLES
+
+PROJECT1_GUID = 'R0001_1kg'
+PROJECT2_GUID = 'R0003_test'
+ADD_DATA_UPDATED_ANVIL_VARIABLES = {
+    "key": "AnVIL_WES",
+    "value": json.dumps({
+        "active_projects": [PROJECT1_GUID],
+        "vcf_path": "gs://test_bucket/test_path.vcf",
+        "project_path": "gs://seqr-datasets/v02/GRCh37/AnVIL_WES/{guid}/v1".format(guid=PROJECT1_GUID),
+        "projects_to_run": [PROJECT1_GUID] })
+}
+ADD_DATA_UPDATE_DAG_TASKS_RESP = {
+            "tasks": [
+                {
+                    "task_id": "create_dataproc_cluster",
+                },
+                {
+                    "task_id": f"pyspark_compute_project_{PROJECT1_GUID}",
+                },
+                {
+                    "task_id": "pyspark_compute_variants_AnVIL_WES",
+                },
+                {
+                    "task_id": f"pyspark_export_project_{PROJECT1_GUID}",
+                },
+                {
+                    "task_id": "scale_dataproc_cluster",
+                },
+                {
+                    "task_id": f"skip_compute_project_subset_{PROJECT1_GUID}",
+                }
+                ],
+            "total_entries": 6
+        }
+
+AIRTABLE_TRACKING_URL = f'{MOCK_AIRTABLE_URL}/appUelDNM3BnWaR7M/AnVIL%20Seqr%20Loading%20Requests%20Tracking'
+
+
+@mock.patch('seqr.views.apis.anvil_workspace_api.id_token.fetch_id_token', lambda *args: MOCK_TOKEN)
+@mock.patch('seqr.views.utils.airtable_utils.AIRTABLE_API_KEY', MOCK_AIRTABLE_KEY)
+@mock.patch('seqr.views.utils.airtable_utils.AIRTABLE_URL', MOCK_AIRTABLE_URL)
+@mock.patch('seqr.views.apis.anvil_workspace_api.AIRFLOW_WEBSERVER_URL', MOCK_AIRFLOW_URL)
+@mock.patch('seqr.views.apis.anvil_workspace_api.BASE_URL', 'http://testserver/')
+@mock.patch('seqr.views.utils.permissions_utils.logger')
+@mock.patch('seqr.views.utils.airtable_utils.logger')
+@mock.patch('seqr.views.apis.anvil_workspace_api.load_uploaded_file')
+@mock.patch('seqr.views.apis.anvil_workspace_api.safe_post_to_slack')
+@mock.patch('seqr.views.apis.anvil_workspace_api.mv_file_to_gs')
+@mock.patch('seqr.views.apis.anvil_workspace_api.tempfile.NamedTemporaryFile')
+@mock.patch('seqr.views.apis.anvil_workspace_api.logger')
+@mock.patch('seqr.views.apis.anvil_workspace_api.datetime')
+@mock.patch('seqr.views.apis.anvil_workspace_api.send_html_email')
+class LoadAnvilDataAPITest(AnvilAuthenticationTestCase):
+    fixtures = ['users', 'social_auth', '1kg_project']
+
+    @staticmethod
+    def _setup_dag_trigger(responses, update_dag_tasks_resp):
         # Set up api responses
-        airtable_tracking_url = f'{MOCK_AIRTABLE_URL}/appUelDNM3BnWaR7M/AnVIL%20Seqr%20Loading%20Requests%20Tracking'
-        responses.add(responses.POST, airtable_tracking_url, status=400)
+        responses.add(responses.POST, AIRTABLE_TRACKING_URL, status=400)
```

Comment:
There is actually another use of the constant. But since that one is for result assertion and it should be replaced with a literal string, so I remove the constant.